### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3353]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-split-button

### Expected behavior

The Accessible name for the menubutton part of the split button should be the same as the accessible name for the button element. They can be distinguished by the fact that one will announce "haspopup" or as a menubutton (depending on the platform). 

### Actual behavior

Currently the menubutton part is announced as "more" which (a) does not get translated and (b) isn't sufficiently descriptive to know what is being pressed.

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/split-button/
2. Inspect the overflow button part of the split button
3. Check the Accessible Name of the split button part in dev tools

**Comment by jnurthen** [comment]
Already filed #3305
